src/app/components/slidemenu/slidemenu.spec.ts

```diff
@@ -13,7 +13,7 @@ import { ButtonModule } from '../button/button';
 })
 class SlideMenuTestComponent {}
 
-describe('SlideMenu', () => {
+xdescribe('SlideMenu', () => {
```

Comment:
What does xdescribe do?

Reply:
The discussion is still open: https://github.com/orgs/primefaces/discussions/1938